### PR TITLE
PYSVC-471 add support file

### PR DESCRIPTION
**Pull request by OndrejM**

### CODE REVIEW

**Comment by OndrejM** [comment]
@smillidge @Pandrex247 please review and merge. Only adds a text file to the github repo

File: SUPPORT.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# Payara Support
+
+The [Payara Foundation](http://www.payara.org) is a non-profit foundation behind the development of Payara Server and Payara Micro (the Payara Platform). The Payara Foundation continues to develop the Payara Platform based on the needs of our support customers.
```
Comment (smillidge) [inline SUPPORT.md]: Payara Foundation doesn't have any customer. So combined with the comment below commercial support should be introduced by Payara Services Limited

File: SUPPORT.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+## Questions and Issues
+
+For questions regarding Payara Server or Payara Micro, including configuration, visit the [Payara Forum](http://www.payara.org/forum)
+
+For issues and bug reports with the Payara Platform, raise a new issue with a reproducer on [GitHub](https://github.com/payara/payara/issues).
+
+- - - -
+
+## Payara Support
+
```
Comment (smillidge) [inline SUPPORT.md]: Add something to say commercial support is offered by Payara Services Limited

**Comment by arjantijms** [comment]
@smillidge @OndrejM This has been open for quite some time. If the changes can't be done, shall we close this instead then?

**Pull request merged**